akka-actor-typed-tests/src/test/scala/docs/akka/typed/IntroSpec.scala

```diff
@@ -68,6 +68,29 @@ object IntroSpec {
   }
   //#hello-world-main
 
+  object CustomDispatchersExample {
+    import HelloWorldMain.Start
+    import akka.actor.typed.Props
+
+    //#hello-world-main-with-dispatchers
+    val main: Behavior[Start] =
+      Behaviors.setup { context ⇒
+        val dispatcherPath = "akka.actor.default-blocking-io-dispatcher"
+
+        val greeterProps = Props.empty.withDispatcherFromConfig(dispatcherPath)
```

Comment:
I think `DispatcherSelector.withDispatcherFromConfig(...)` just like below is cleaner than going via `Props.empty`

Reply:
I agree. Even though there might be a reason for having two ways of doing the same thing we shouldn't show both. If it's only the dispatcher then `DispatcherSelector.fromConfig` is probably cleanest.